src/chatbot/bot.py

```diff
@@ -75,6 +76,32 @@ async def show_score(ctx: Context) -> None:
     await ctx.send(f"Your social score is currently {user_to_lookup.social_score}")
 
 
+@bot.command()
+@is_in_channel(set(["bot-commands"]))
+async def register_score(
+    ctx: Context, emoji: Union[PartialEmoji, str], score: int
+) -> None:
+    if isinstance(emoji, PartialEmoji):
+        emoji_id = str(emoji.id)
+    else:
+        emoji_id = emoji
+    logger.info("Registering emoji_id %s with score %d", emoji_id, score)
+    emoji_score: EmojiScore = await EmojiScore.objects.async_get(emoji_id=emoji_id)
```

Comment:
It looks like you want a `update_or_create` here? Probably worth adding it to the async manager stuff.

https://docs.djangoproject.com/en/3.2/ref/models/querysets/#update-or-create